ocean_lib/models/test/test_data_nft.py

```diff
@@ -714,80 +619,19 @@ def test_fail_create_datatoken(
     )
     data_nft.addToCreateERC20List(consumer_wallet.address, {"from": publisher_wallet})
 
-    # Should fail to create a specific ERC20 Template if the index is ZERO
-    with pytest.raises(Exception, match="Template index doesnt exist"):
-        data_nft.create_datatoken(
-            DatatokenArguments(
-                template_index=0,
-                name="DT1",
-                symbol="DT1Symbol",
-            ),
-            {"from": consumer_wallet},
-        )
-
-    # Should fail to create a specific ERC20 Template if the index doesn't exist
-    with pytest.raises(Exception, match="Template index doesnt exist"):
-        data_nft.create_datatoken(
-            DatatokenArguments(
-                template_index=3,
-                name="DT1",
-                symbol="DT1Symbol",
-            ),
-            {"from": consumer_wallet},
-        )
-
-    # Should fail to create a specific ERC20 Template if the user is not added on the ERC20 deployers list
-    assert data_nft.getPermissions(another_consumer_wallet.address)[1] is False
-    with pytest.raises(Exception, match="NOT ERC20DEPLOYER_ROLE"):
-        data_nft.create_datatoken(
-            DatatokenArguments(
-                template_index=1,
-                name="DT1",
-                symbol="DT1Symbol",
-            ),
-            {"from": another_consumer_wallet},
-        )
-
-
-@pytest.mark.unit
-def test_datatoken_cap(publisher_wallet, consumer_wallet, data_nft_factory):
-    # create NFT with ERC20
-    with pytest.raises(Exception, match="Cap is needed for Datatoken Enterprise"):
```

Comment:
this is our in-house Exception, it has nothing to do with the contracts. Keep it, please